__tests__/api.test.js

```diff
@@ -96,9 +94,46 @@ describe('Articles', ()=>{
                     });
             })
         })
-
+        test("the articles should be sorted by date in descending order", ()=>{
+            return request(app)
+            .get("/api/articles")
+            .expect(200)
+            .then((response) => {
+                const articles = response.body.articles;
+                expect(Array.isArray(articles)).toBe(true);
+                expect(articles).toBeSorted({ created_at: 'id' })
+            })
+        })
     })
+})
 
+describe ("/api/articles/:article_id/comments", ()=>{
+    test('GET:200 sends an array of comments for the given article_id for each comment', () => {
+        return request(app)
+            .get('/api/articles/1/comments')
+            .expect(200)
+            .then((response) => {
+            expect(response.body.comments.length).toBe(11);
+            response.body.comments.forEach((comment) => {
+                expect(typeof comment.comment_id).toBe('number');
+                expect(typeof comment.votes).toBe('number');
+                expect(typeof comment.created_at).toBe('string');
+                expect(typeof comment.author).toBe('string');
+                expect(typeof comment.body).toBe('string');
+                expect(typeof comment.article_id).toBe('number');
+                expect(comment.article_id).toBe(1);
+            });
+            });
+        });
+    test("comments should be returned with the most recent comments first", ()=>{
+        return request(app)
+        .get('/api/articles/1/comments')
+        .expect(200)
+        .then((response) => {
+            const comments = response.body.comments;
+            expect(comments).toBeSorted({ created_at: 'id' })
+        })
+    })
```

Comment:
you need tests for the following: 

 - 404: valid but none existing article_id
 - 400: invalid article_id type
 - 200: when article_id is valid but the article has no comments.